### PR TITLE
Enable Search Dev Tools for 25% of sites and ALL non-prod sites

### PR DESCRIPTION
**Pull request by rinatkhaziev**
## Description

This PR enables Search Dev Tools for 25% of sites and ALL non-prod sites.

### Plugin Updated: Search Dev Tools

We enabled Search Dev Tools on 25% of sites and all non-production sites.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [ ] I've created a changelog description that aligns with the provided examples.

### CODE REVIEW

**Review by netsuso — APPROVED** [review]
Let's go! :)

**Comment by pauarge** [comment]
PR looks good to me, but I seem to recall similar constant definition on the `.secrets`. Will we remove the code from there once this is merged?

**Comment by rinatkhaziev** [comment]
@pauarge that's correct.

Seems I need to fix some tests here before this can be merged.

File: 000-vip-init.php [hunk 1]
```diff
@@ -212,6 +216,9 @@
 	require_once( __DIR__ . '/search/search.php' );
 }
 
+
+
+
```
Comment (pauarge) [inline 000-vip-init.php:221]: Not sure we need to add those line breaks.

Comment (rinatkhaziev) [reply 000-vip-init.php:221]: Yeah, a leftover from moving the lines. Not sure why linter didn't catch it.

**Review by pauarge — APPROVED** [review]

**Comment by rinatkhaziev** [comment]
r1625-stacks

**Pull request merged**